<HTML>
<HEAD>
<meta charset="UTF-8">
<title>CustomValidator.validate - vertx</title>
<link rel="stylesheet" href="../../../style.css">
</HEAD>
<BODY>
<a href="../../index.html">vertx</a>&nbsp;/&nbsp;<a href="../index.html">io.vertx.reactivex.ext.web.api.validation</a>&nbsp;/&nbsp;<a href="index.html">CustomValidator</a>&nbsp;/&nbsp;<a href=".">validate</a><br/>
<br/>
<h1>validate</h1>
<a name="io.vertx.reactivex.ext.web.api.validation.CustomValidator$validate(io.vertx.reactivex.ext.web.RoutingContext)"></a>
<code><span class="keyword">open</span> <span class="keyword">fun </span><span class="identifier">validate</span><span class="symbol">(</span><span class="identifier" id="io.vertx.reactivex.ext.web.api.validation.CustomValidator$validate(io.vertx.reactivex.ext.web.RoutingContext)/routingContext">routingContext</span><span class="symbol">:</span>&nbsp;<span class="identifier">RoutingContext</span><span class="symbol">)</span><span class="symbol">: </span><a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-unit/index.html"><span class="identifier">Unit</span></a></code>
<p>This function have to be <strong>synchronous</strong>. It doesn't return nothing if validation succedes, otherwise it throws ValidationException.  <strong>Don't call routingContext.next() or routingContext.fail() from this function</strong></p>
<h3>Parameters</h3>
<p><a name="routingContext"></a>
<code>routingContext</code> - the actual routing context</p>
</BODY>
</HTML>
